pkg/networkmanager/interfaces.js

```diff
@@ -829,8 +875,9 @@
                 .finally(pop_refresh);
     }
 
-    function handle_updated(obj) {
-        refresh_settings(obj);
+    function handle_updated(obj, things) {
+        if (things.length > 0)
+            refresh_settings(obj);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19024-20230704-110812-1ea465d9-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)